hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/test/java/org/apache/hadoop/yarn/server/federation/policies/router/TestPriorityRouterPolicy.java

```diff
@@ -54,10 +55,11 @@ public void setUp() throws Exception {
 
       // with 5% omit a subcluster
       if (getRand().nextFloat() < 0.95f || i == 5) {
-        SubClusterInfo sci = mock(SubClusterInfo.class);
-        when(sci.getState()).thenReturn(SubClusterState.SC_RUNNING);
-        when(sci.getSubClusterId()).thenReturn(sc.toId());
-        getActiveSubclusters().put(sc.toId(), sci);
+        long now = Time.now();
+        SubClusterInfo federationSubClusterInfo =
```

Comment:
Not sure the indetation is correct.
Let's do:
```
SubClusterInfo federationSubClusterInfo = SubClusterInfo.newInstance(
    sc.toId(), "dns1:80", "dns1:81", "dns1:82", "dns1:83",
    now - 1000, SubClusterState.SC_RUNNING, now - 2000, generateClusterMetricsInfo(i));
```

Reply:
I will fix it.